src/mainboard/clevo/adl-p/spd/samsung-P4AAF165WA-BCWDE.spd.hex

```diff
@@ -0,0 +1,33 @@
+# Samsung P4AAF165WA-BCWDE
```

Comment:
NS70PU does not have soldered memory, so this file can be removed

Reply:
Removed.